app/resources/api/v1/set_resource.rb

```diff
@@ -7,12 +7,18 @@ class SetResource < JSONAPI::Resource
       has_many :materials, class_name: 'Material', relation_name: :materials, acts_as_set: true
 
       filter :name
+      filter :locked
       # http://localhost:3000/api/v1/sets?filter[owner_id]=guest
       filter :owner_id, apply: -> (records, value, _options) {
         return records.none if value.nil?
         records.where(owner_id: value)
       }
 
+      def self.default_sort
+        [{field: 'created_at', direction: :desc}]
+      end
```

Comment:
What are the implications of changing the default sort? Is there anywhere in Aker that we've been relying on a different order by default?

Reply:
The reason behind this change is to keep consistency in the way we make requests. At now in Set Shaper we are displaying the sets in pages without providing explicitly the filter for pagination. If we want to display the last 25 created sets for a user it would be strange if we make mandatory to provide the filter for sorting but not the one for pagination.
Also, the default sort uses the id for the table; because the id is an uuid it seems more meaningful to sort by the creation date, or by some other criteria.

Reply:
This change introduces inconsistency in our APIs. Why only sort Sets by creation date, and not Materials, Permissions, Studies, etc. as well? 

Why is it be strange to add a sort parameter to the request to get a user's Sets in the Set Shaper? e.g. `sets?sort=-created_at&filter[owner_id]=cs24` vs `sets?filter[owner_id]=cs24`